packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -43,15 +41,28 @@ const styles = (theme: ThemeType): JssStyles => ({
     maxWidth: 252,
```

Comment:
Could you maybe add more whitespace to the top of admin notice? It currently looks kinda bad to me, and even though it only affects admins, I'd rather not train my eyes to ignore bad UI.

Reply:
Yup, increased the margin